worlds/fe8/__init__.py

```diff
@@ -189,6 +190,14 @@ def create_items(self) -> None:
                     else ItemClassification.useful,
                 )
             )
+        
+        for f in FILLER_ITEMS:
```

Comment:
We should instead select one item for every unfilled location (as well as discard items if we happen to overflow locations)